net/openrsync/Portfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        kristapsdz openrsync a257c0f495af2b5ee6b41efc6724850a445f87ed
+version             20250127
+revision            0
+
+categories          net
+license             ISC
+maintainers         {@artkiver gmail.com:artkiver} openmaintainer
+
+description         a re-implementation of rsync from OpenBSD developers
+long_description    rsync has specific running modes for the super-user. \
+                    It also pumps arbitrary data from the network onto \
+                    your file-system. openrsync is about 10 000 lines of C \
+                    code: do you trust me not to make mistakes?
+
+homepage            https://openrsync.org/
+
+checksums           rmd160  244feafe48bf2095d366aabd6de8416021ea6bca \
+                    sha256  e2def3d03bcb61a584c1f878dd66da4dc7ca889652689096dc0cd0f5bd7371fa \
+                    size    152816
```

Comment:
If you add a build dependency on `coreutils` (path-style, there is a -devel subport) and add `${prefix}/libexec/gnubin` in front of the PATH in configure.env, it fixes the bug with `mktemp`.